docs/porting_guide.rst

```diff
@@ -104,7 +115,11 @@ User will need to implement the above platform specific remoteproc driver functi
 Platform Specific Porting to Use RPMsg
 **************************************
 
-RPMsg in OpenAMP implementation uses VirtIO to manage the shared buffers. OpenAMP library provides remoteproc VirtIO backend implementation. You don't have to use remoteproc backend. You can implement your VirtIO backend with the VirtIO and RPMsg implementation in OpenAMP. If you want to implement your own VirtIO backend, you can refer to the [remoteproc VirtIO backend implementation]: https://github.com/OpenAMP/open-amp/blob/master/lib/remoteproc/remoteproc_virtio.c
+RPMsg in OpenAMP implementation uses VirtIO to manage the shared buffers. OpenAMP library provides
+remoteproc VirtIO backend implementation. You don't have to use remoteproc backend. You can
+implement your VirtIO backend with the VirtIO and RPMsg implementation in OpenAMP. If you want to
+implement your own VirtIO backend, you can refer to the [remoteproc VirtIO backend implementation]:
```

Comment:
I believe this was supposed to be a link pointing to https://github.com/OpenAMP/open-amp/blob/master/lib/remoteproc/remoteproc_virtio.c, but the rst link format is not used.

Reply:
done